src/features/vote/RevealPhase.tsx

```diff
@@ -204,7 +204,7 @@ const RevealPhase: FC<Props> = ({
                                 }
                               })
                               .catch((err) => {
-                                console.log("err in snapshot", err);
+                                console.error("err in snapshot", err);
```

Comment:
Can we add err.stack to the printout? I've found it's very hard to locate the line where the error is happening without this. If there's stuff in the error you want to keep that's not in the stack, maybe we could print the stack separately?

Reply:
logging the entire error object will print the stack out without explicitly needing to print err.stack

Reply:
My understanding (that could be wrong) is that these are different. I've seen this in node.

Stringifying an error gives you the message + "name": https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/toString.

Printing out the stack gives you the line where the error was thrown: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/Stack.